vortex-scalar/src/scalarvalue/mod.rs

```diff
@@ -36,37 +38,31 @@ pub(crate) enum InnerScalarValue {
     List(Arc<[ScalarValue]>),
 }
 
-#[cfg(feature = "flatbuffers")]
 impl ScalarValue {
-    pub fn to_flexbytes<B: Default + for<'a> Extend<&'a u8>>(&self) -> B {
-        use serde::Serialize;
-        use vortex_error::VortexExpect;
-
-        let mut ser = flexbuffers::FlexbufferSerializer::new();
-        self.serialize(&mut ser)
-            .vortex_expect("Failed to serialize ScalarValue");
-        let view = ser.view();
-
-        let mut buf = B::default();
-        buf.extend(view);
-        buf
+    pub fn to_protobytes(&self) -> ByteBuffer {
```

Comment:
This shouldn't return a ByteBuffer since alignment isn't required. Either use tokio::Bytes, or Vec<u8> (I think the latter, since it can be zero-copy to Bytes, but not the other way around)

Reply:
Ok, I restored the old behaviour since prost has tokio::Bytes integration